integration_tests/ctesque/src/androidTest/java/android/app/ActivityInstrTest.java

```diff
@@ -113,4 +115,44 @@ public void audioManager_activityContextEnabled() {
       System.setProperty("robolectric.createActivityContexts", originalProperty);
     }
   }
+
+  @Test
+  public void accountManager_activityContextEnabled() {
+    String originalProperty = System.getProperty("robolectric.createActivityContexts", "");
+    System.setProperty("robolectric.createActivityContexts", "true");
+    try {
+      AccountManager applicationAccountManager =
+          (AccountManager)
+              ApplicationProvider.getApplicationContext().getSystemService(Context.ACCOUNT_SERVICE);
+      Log.d("Test", "Retrieved applicationAccountManager instance");
+
+      try (ActivityScenario<TestActivity> scenario = ActivityScenario.launch(TestActivity.class)) {
+        scenario.onActivity(
+            activity -> {
+              AccountManager activityAccountManager =
+                  (AccountManager) activity.getSystemService(Context.ACCOUNT_SERVICE);
+              assertThat(applicationAccountManager).isNotSameInstanceAs(activityAccountManager);
+              Log.d("TestActivity", "Retrieved activityAccountManager instance");
+
+              // Example operation: Retrieve accounts of a specific type
+              Account[] applicationAccounts =
+                  applicationAccountManager.getAccountsByType("com.example.account_type");
+              Account[] activityAccounts =
+                  activityAccountManager.getAccountsByType("com.example.account_type");
+
+              Log.d("TestActivity", "applicationAccounts length: " + applicationAccounts.length);
+              Log.d("TestActivity", "activityAccounts length: " + activityAccounts.length);
+
+              // Add assertions to validate behavior
+              assertThat(activityAccounts.length).isEqualTo(applicationAccounts.length);
```

Comment:
Can this just be: `assertThat(activityAccounts).isEqualTo(applicationAccounts)`?